### PR TITLE
Make the tests run again

### PR DESCRIPTION
**Pull request by KyleW**
This is a quick fix to get the test running again. To make it work, I had to disable browserify-istanbul and ignore two of the sets of tests. It's not ideal and I'll be fixing the remainder of the tests soon. But this makes the gulp karma and related tasks (gulp build) run without erroring out. I wanted to share that asap in case it was blocking others.

### CODE REVIEW

**Comment by KyleW** [comment]
@jnmarcus - Take a look.

File: app/components/homePageModule/HomePageController.js [hunk 1]
```diff
@@ -3,10 +3,12 @@
 // Controller naming conventions should start with an uppercase letter
 // function HomePageController($scope, $log, $http, TestFactory1, disclosureApi) {
 function HomePageController($scope, $log, $http, TestFactory1) {
+  var localitiesList;
+
   $scope.searchBarEnabled = false;
   $scope.testVar = 'We are up and running using a required module!';
   //$scope.searchResults = [];
-  var localitiesList = {};
+  localitiesList = {};
```
Comment (jnmarcus) [inline app/components/homePageModule/HomePageController.js]: @KyleW you can pull this variable.  The controllers aren't sharing scopes so if it's not being used in the controller where it's specified, it's not being used at all.

File: app/components/appMainModule/examplePage1Module/testService1.js [hunk 2]
```diff
@@ -57,7 +57,7 @@
         .then(getPageDataComplete)
         .catch(getPageDataFailed);
 
-      function getPageDataComplete(data, status, headers, config) {
+      function getPageDataComplete(data) {
```
Comment (jnmarcus) [inline app/components/appMainModule/examplePage1Module/testService1.js]: @KyleW we could probably delete this function here. It's referenced in line 85 for the factory to access, so it shouldn't cause any problems.

**Comment by jnmarcus** [comment]
@KyleW Thanks for this! :simple_smile: I made a couple comments inline that would be great to fix before you merge, but then it looks good to me.

I'd prefer to keep line 85 in testService.js as it was (ie with the (status, header, config) ), only because everything in the example page folder is meant for people to use as an example, including people that are new to JavaScript and frontend development in general (which has been a repeated request by people in various brigades).  Up to you though if you want to add it back in or not.

Other than that, it looks good to me 😃

File: app/components/homePageModule/HomePageController.js (hunk 1 above)
Comment (KyleW) [inline app/components/homePageModule/HomePageController.js]: gone!

File: app/components/appMainModule/examplePage1Module/testService1.js (hunk 2 above)
Comment (KyleW) [inline app/components/appMainModule/examplePage1Module/testService1.js]: I'm not sure I follow. If it's used on line 85, should I leave it there? I'll leave it for now and we can always delete it later. The point was mostly to make the linter happy and get the tests to pass.

**Comment by KyleW** [comment]
Made the changes based on the comments above.  @jnmarcus - Any objections to me merging this? I believe it will cause whatever is on develop to be deployed since the tests will now pass.

**Comment by jnmarcus** [comment]
@KyleW thanks for this! this looks great and merge-able to me! 👍

**Pull request merged**